recipe/meta.yaml

```diff
@@ -15,14 +15,15 @@ source:
 
 build:
   number: 1
```

Comment:
I did not bump the build number because we are just adding the other two architectures.

If you do want to bump it, then it might make sense to update the `license` field and add the `run_exports` metadata.

@conda-forge/libpcap  this is ready for review.

Reply:
But things like the OSX build image changed.

And likely unpinned dependencies changed.

Isn't it good to change the number in case?

Reply:
good point!